### PR TITLE
Scoreboard judge error bug fix

### PR DESCRIPTION
**Pull request by Changron**
1. won't be calculated in scoreboard if submission is Judge Error
2. fix contest clock and add days
   
3. add contest frozen hint

### CODE REVIEW

File: contest/static/contest/js/contest.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ function checkTime(i) {
 function getRestTime() {
```
Comment (nickelcheng) [inline contest/static/contest/js/contest.js]: This function is quite same with that in [this file](https://github.com/Changron/NTHUOJ_web/blob/dev/index/static/js/contestTimer.js)
Could you please reuse it?

**Comment by nickelcheng** [comment]
This PR is good except for the comment.
@CHIAHSINHOU please test it

File: contest/static/contest/js/contest.js (hunk 1 above)
Comment (Changron) [inline contest/static/contest/js/contest.js]: some parts are same, but there are something different.
Contest has to show" Contest Not Started Yet" and "Contest Ended",
but in index it doesn't.
It is not quite easy to use same code

Comment (nickelcheng) [inline contest/static/contest/js/contest.js]: That's ok, just keep it.

**Comment by nickelcheng** [comment]
please merge the latest code

**Comment by Changron** [comment]
merged

**Comment by CHIAHSINHOU** [comment]
what happened?

**Comment by Changron** [comment]
fixed

**Comment by CHIAHSINHOU** [comment]
This PR is ok for me.

**Comment by bruce3557** [comment]
+1

**Comment by nickelcheng** [comment]
going to merge

**Pull request merged**